redirector/common.py

```diff
@@ -80,16 +81,16 @@ def get_ddb_table():
     return boto3.resource("dynamodb").Table(MANIFEST_TABLE_NAME)
 
 
-def get_file_pathname(file):
+def get_filepath(file):
     # want to start the path after "All Files/<BoxFolderName>/"
-    file_path_collection = file.path_collection
-    start_index = [e.id for e in file_path_collection["entries"]].index(
+    filepath_collection = file.path_collection
+    start_index = [e.id for e in filepath_collection["entries"]].index(
         BOX_FOLDER_ID
     ) + 1
-    file_path_names = [
-        fp.get().name for fp in file_path_collection["entries"][start_index:]
+    filepath = [
```

Comment:
filepaths?

Reply:
the reason I didn't call it `filepaths` is because each entry is really one folder in the path to a file. So it's not really `paths` plural IMO. It's just the path to a file in list format, which then gets converted to and returned as a string.

Reply:
addressed in a5a427d